cranelift-codegen/meta/src/gen_legalizer.rs

```diff
@@ -162,10 +159,14 @@ fn unwrap_inst(
             let name = var_pool
                 .get(apply.args[i].maybe_var().expect("vararg without name"))
                 .name;
-
-            // Above name is set to an `Vec` representing the varargs. However it is expected to be
-            // `&[Value]` below, so we borrow it.
-            fmtln!(fmt, "let {} = &{};", name, name);
+            let n = inst
+                .imm_opnums
+                .iter()
+                .chain(inst.value_opnums.iter())
+                .max()
+                .map(|n| *n)
```

Comment:
FWIW:

```suggestion
                .cloned()
```

Reply:
```suggestion
                .copied()
```

https://doc.rust-lang.org/stable/core/iter/trait.Iterator.html#method.copied stable since 1.36.0